pass-model/src/main/java/org/dataconservancy/pass/model/Policy.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2018 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataconservancy.pass.model;
+
+import java.net.URI;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Describes a Policy. Policies determine the rules that need to be followed by a Submission.

```

Comment:
I presume that when PASS has a policy service or engine of some kind that this information would need to be reconciled there?  That is, long-term will it be the responsibility of the java client to provide the modeling for policy?

Reply:
to be determined... this just reflects the OA-PASS/pass-data-model for now.